namespaces/_default_.yml

```diff
@@ -190,7 +190,12 @@ _index_type_:
         
     - name: viaq_index_name
       type: keyword
-      example: project.my-cool-project-in-lab04.748e92c2-70d7-11e9-b387-000d3af2d83b.2019.05.09
+      example: container.app-000001
```

Comment:
i think the write alias per the enhancement would be something like `container.app-write`

Reply:
Good catch. Updated.